timemachine/fe/free_energy.py

```diff
@@ -374,15 +378,16 @@ def sample(
 
     assert np.all(np.isfinite(ctxt.get_x_t())), "Equilibration resulted in a nan"
 
-    def run_production_steps(n_steps: int) -> Tuple[NDArray, NDArray]:
+    def run_production_steps(n_steps: int) -> Tuple[NDArray, NDArray, NDArray]:
         coords, boxes = ctxt.multiple_steps(
             n_steps=n_steps,
             store_x_interval=md_params.steps_per_frame,
         )
+        final_velocities = ctxt.get_v_t()
```

Comment:
sanity check, does `ctxt.get_x_t()` match `coords[-1]`? i.e. making sure that the last frame of coords are in sync with the velocities.

Reply:
Assertion added in [d151e39](https://github.com/proteneer/timemachine/pull/1142/commits/d151e39490322bf88df7f07989cad682fa6fa785)